### DISCUSSION TITLE
How to back up -- Zavala doesn't show up in iCloud drive

### DISCUSSION THREAD

**Discussion by questionauthority** [vincode-io/Zavala#258] [Support] [answered]
Hello, thank you for this splendid, free tool.  :) I'm a new Zavala user and a relatively new macbook user.  My question is how to back up.  

My understanding is that the answer is iCloud.  However, even though I have iCloud drive enabled and can see files from my desktop, I don't see any of my Zavala outlines in it.  

Thanks in advance for any tips you're able to provide.

**Comment by ghost** [comment]
Well, to have outlines synced to iCloud, you need turn on "Enable iCloud" options in the app settings ("Accounts" section). Then they'll show up in iCloud. 

To have a full backup you can expert outline as an archive, details here: https://zavala.vincode.io/help/Shortcut_Examples.md/

**Reply by questionauthority** [reply]
Thanks!  I already enabled iCloud before posting this.

**Reply by ghost** [reply]
Are you sure you've enabled it in the app? I understood from your post that you've enabled iCloud in the system settings, as a storage. There's another switch in the app itself which enables iCloud for this particular app.

**Reply by questionauthority** [reply]
Yes, I'm sure I enabled iCloud in the app as well.  Maybe it needs 24 hours to sync or something?

**Reply by questionauthority** [reply]
Thanks for the link!  I clicked Archive Account and then Get Shortcut followed by Open Link then Add Shortcut.  Then I open the Archive Account folder and am clueless.  I'm wondering if this app is above my skill level.  Is Zavala not for people with average tech skills?

**Reply by vincode-io** [reply]
Zavala is intended to be simple and easy to use with advanced functionality possible using Shortcuts. Since you are new to the Mac, you aren't yet familiar with Shortcuts which is causing some confusion. You can take the time to learn about Shortcuts on the Mac or just use Zavala without them.

**Comment by vincode-io** [comment, marked as answer]
iCloud itself isn't a good backup solution. If something gets deleted from iCloud it is truly gone and you can't get it back. As @mikezelazny mentioned, you want to use the Archive Account shortcut if you aren't using TimeMachine or some other backup solution.

One thing to keep in mind, is that Zavala is a shoebox app, like Music.app or Notes.app. That means that it has its own internal database that isn't human readable and you have to access your Outlines through Zavala unless you export them. You will never see unexported outlines in iCloud Drive.

**Reply by questionauthority** [reply]
Thanks for explaining that; that helps.  :)

**Reply by questionauthority** [reply]
Now I'm seeing an error message that says "CloudKit Syncing Error," but I guess that's irrelevant because I'm gonna try the Archive Account feature.

**Comment by questionauthority** [comment]
Thanks for clarifying that.  I do love its simplicity.

El lun, 4 nov 2024 a la(s) 2:36 p.m., Maurice Parker (
***@***.***) escribió:

> Zavala is intended to be simple and easy to use with advanced
> functionality possible using Shortcuts. Since you are new to the Mac, you
> aren't yet familiar with Shortcuts which is causing some confusion. You can
> take the time to learn about Shortcuts on the Mac or just use Zavala
> without them.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vincode-io/Zavala/discussions/258#discussioncomment-11143476>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMULRMWB7QUP7IIY7N3Y573Z65Z4NAVCNFSM6AAAAABRDDJV5OVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMJUGM2DONQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Subscribe to Mirrors, Signal, Blind Spot
<https://mooreamaguire.substack.com/>, where I summarize the latest
research that's not covered by corporate or independent media.
